### PR TITLE
Capi docs

### PR DESCRIPTION
**Pull request by evilnick**
Linked some generic pages to the Capi nav
Fixed some linting issues
Reworked some links so they work relatively
minor corrections

### CODE REVIEW

**Review by eaudetcobello — APPROVED** [review]

File: docs/src/charm/howto/charm.md [hunk 1]
```diff
@@ -48,8 +48,8 @@ will find the latest features but you may also experience instability.
 
 ## Bootstrap the cluster
 
-Installing the k8s charm sets up all the parts required to run Kubernetes. One may
-watch it progress using juju status:
+Installing the `k8s` charm sets up all the parts required to run Kubernetes.
+You can watch it progress using juju status:
```
Comment (louiseschmidtgen) [inline docs/src/charm/howto/charm.md:52]:
```suggestion
You can watch the installation progress using juju status:
```

File: docs/src/charm/howto/install-lxd.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Installing to localhost/LXD
 
 The main [install instructions][install] work for most circumstances when you
```
Comment (louiseschmidtgen) [inline docs/src/charm/howto/install-lxd.md:3]:
```suggestion
The main [install instructions][install] cover most situations for installing {{product}} using a charm. However, there are a few scenarios that require additional steps. These include:
```

File: docs/src/charm/howto/install-lxd.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
-want to install Canonical Kubernetes from a charm. There are a couple of
+want to install {{product}} from a charm. There are a couple of
 scenarios which require some extra steps however. These are:
```
Comment (louiseschmidtgen) [inline docs/src/charm/howto/install-lxd.md:5]:
```suggestion
```

File: docs/src/index.md [hunk 4]
```diff
@@ -71,7 +71,16 @@ Our tutorials, How To guides and other pages will explain how to install,
  configure and use the {{product}} 'k8s' charm.
 ```
 
+
+```{grid-item-card}
+:link: capi/
+### [Deploy K8s using Cluster API ›](capi/index)
+^^^
+Our tutorials, guides and other pages will explain how to install,
```
Comment (louiseschmidtgen) [inline docs/src/index.md:79]:
```suggestion
Our tutorials, guides and explanation pages will explain how to install,
```
The use of other seems a bit vague here.

File: docs/src/snap/explanation/about.md [hunk 5]
```diff
@@ -34,6 +34,8 @@ be: run it in several local containers or VMs for example, or use it on
 public/private cloud instances. 
 For deploying with [Juju], a machine [charm] to deploy
 the snap is also available.
+For deployment using Cluster API, additional resources have also been
```
Comment (bschimke95) [inline docs/src/snap/explanation/about.md:37]: For deployment using Cluster API, a Bootstrap and Control-plane provider have also been created.

**Review by louiseschmidtgen — APPROVED** [review]
A few minor suggestions. Thanks for adding all the missing bits and pieces @evilnick!

**Review by bschimke95 — APPROVED** [review]
Two nits, great work

**Pull request merged**